state_solver/suspension.py

```diff
@@ -13,43 +13,53 @@
 
 class Suspension():
     """ Handles suspension vehicle forces """
-    def __init__(self, params:vehicle_params.BaseVehicle, logger:engine.Logger):
+    def __init__(self, params, logger:Logger):
         self.logger = logger
-        self.params:vehicle_params.BaseVehicle = params
-        self.__tires = engine.Tires(params)
+        self.params = params
+        self.__tires = Tires(params)
 
 
-    def get_loads(self, vehicle_velocity:np.array, yaw_rate:float, steered_angle:float, roll:float, pitch:float, heave:float):
-        forces, moments = np.array([0, 0, 0]), np.array([0, 0, 0])
+    def get_loads(self, vehicle_velocity:np.array, yaw_rate:float, steered_angle:float, roll:float, pitch:float, heave:float, slip_ratios:list):
+        veh_forces, veh_moments, wheel_speeds, wheel_torques = np.array([0, 0, 0]), np.array([0, 0, 0]), np.array([]), np.array([])
         
         # tire output forces as a function of inclination angle, slip angle, and normal force
         for tire_name, tire in self.__tires.items():
             
             ### ~~~ Normal Force Calculation ~~~ ###
             normal_force = self.__get_tire_normal_load(tire_name, tire, heave, pitch, roll)
-            
+
             ### ~~~ Slip Angle Calculation ~~~ ###
-            tire_velocity = vehicle_velocity + np.cross(np.array([0, 0, yaw_rate]), tire.position) # in IMF
+            tire_IMF_velocity = vehicle_velocity + np.cross(np.array([0, 0, yaw_rate]), tire.position) # in IMF
             steering_toe_slip = tire.steering_induced_slip(steered_angle)
-            slip_angle = np.arctan2(tire_velocity[1], tire_velocity[0]) +  steering_toe_slip# f(steered angle, body slip, yaw rate)
+            slip_angle = -np.arctan2(tire_IMF_velocity[1], tire_IMF_velocity[0]) +  steering_toe_slip# f(steered angle, body slip, yaw rate)
```

Comment:
Why the weird negative?

Reply:
Long explanation. Short answer, is correct if you write out the dynamics on paper and fixed positive long accel issues

Reply:
uhh ok can you add documentation? this is counter intuitive to me. maybe fix this in transforms PR

Reply:
Yeah lets wrap this into transforms PR, definitely think this would be absolved in that